bottlecap/src/secrets/decrypt.rs

```diff
@@ -53,7 +53,18 @@ pub async fn resolve_secrets(config: Arc<Config>, aws_config: &mut AwsConfig) ->
             }
 
             let decrypted_key = if config.kms_api_key.is_empty() {
-                decrypt_aws_sm(&client, config.api_key_secret_arn.clone(), aws_config).await
+                let secret_region = config
```

Comment:
Might be good to have a note that secrets might not be from the same region

Reply:
can do that